### PR TITLE
Importing tt_001 clut data

### PR DESCRIPTION
**Pull request by Onenutmcgee**
Again, data is shared with tt_000, so I merged that into a shared header.

### CODE REVIEW

**Review by bismurphy — APPROVED** [review]
Looks good, nicely done. Great to be able to share code between overlays.

This is totally meaningless, but just from a style point of view "decompile" usually only refers to creating C code for functions. Totally fine though.

**Comment by Onenutmcgee** [comment]
I didn't know what to call it besides decompile.  I agree that is wrong terminology.  I guess maybe importing data sections?

**Comment by bismurphy** [comment]
yeah importing is great; no reason to change this PR or anything, just a terminology nitpick for future, it really doesn't matter though :)

File: src/servant/tt_000/10E8.c [hunk 1]
```diff
@@ -450,12 +448,7 @@ void ServantInit(InitializeMode mode) {
     SpriteParts** spriteBanks;
     Entity* e;
 
-#ifdef VERSION_PC
-    // i exceeds the size of D_80170448
-    const int len = LEN(D_80170448);
```
Comment (sozud) [inline src/servant/tt_000/10E8.c:455]: Why was this removed? This is an out of bounds bug

Comment (Onenutmcgee) [reply src/servant/tt_000/10E8.c:455]: I removed it because if it's there it causes the macOS and linux builds to fail.  I'll add it back in.  Maybe you can help me figure out how to fix

Comment (Onenutmcgee) [reply src/servant/tt_000/10E8.c:455]: I figured it out.  I had removed an extern variable declaration which was what was breaking those builds.

**Comment by sozud** [comment]
Needs a rebase for conflicts from previous PR
